packages/angular_devkit/core/node/experimental/job-registry.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import { Observable, of } from 'rxjs';
+import { JsonValue, experimental as core_experimental, schema } from '../../src';
+import { ModuleNotFoundException, resolve } from '../resolve';
+
+export class NodeModuleJobRegistry<MinimumArgumentValueT extends JsonValue = JsonValue,
+  MinimumInputValueT extends JsonValue = JsonValue,
+  MinimumOutputValueT extends JsonValue = JsonValue,
+> implements core_experimental.jobs.Registry<MinimumArgumentValueT,
+  MinimumInputValueT,
+  MinimumOutputValueT> {
+  public constructor(private _resolveLocal = true, private _resolveGlobal = false) {
+  }
+
+  _resolve(name: string): string | null {
```

Comment:
Why is this a public but prefixed with an underscore?

Reply:
Should be protected.